style.css

```diff
@@ -1,705 +1,705 @@
-  color: #ebebff;
-}
-
-.contact #fname {
-  margin-top: 10%;
-  width: 295px;
-  padding: 12px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
-}
-
-.contact #email {
-  margin-top: 5%;
-  width: 295px;
-  padding: 12px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
-}
-
-.contact #msg {
-  padding: 15px;
-  margin-top: 5%;
-  width: 290px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #b3bac5;
-  padding-bottom: 36%;
-}
-
-.contact .submit {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 12px 16px;
-  position: absolute;
-  width: 125px;
-  height: 30px;
-  top: 4550px;
-  background: #fff;
-  border-radius: 8px;
-}
-
-.contact .submit .tt {
-  text-decoration: none;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #6070ff;
-} */
+/* stylelint-disable declaration-block-no-shorthand-property-overrides */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,700,900');
+
+html {
+  width: 375px;
+  height: 4410px;
+  background-color: #f7f6f6;
+}
+
+header {
+  margin-top: 30px;
+  margin-bottom: 20px;
+  padding-left: 5%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+header a {
+  color: #6070ff;
+  font-family: 'Poppins';
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+nav {
+  margin-left: 65%;
+}
+
+.hero {
+  padding-left: 5%;
+  width: 375px;
+  height: 500px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(/image/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+}
+
+section h1 {
+  color: #172b4d;
+  font-family: 'Poppins';
+  font-weight: 700;
+  line-height: 52px;
+  font-size: 40px;
+}
+
+section p {
+  color: #344563;
+  font-family: 'Poppins';
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  width: 340px;
+}
+
+section a {
+  font-family: 'Poppins';
+  text-decoration: none;
+  color: #6070ff;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.social {
+  margin-left: -40px;
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  height: 20px;
+}
+
+.social li {
+  padding-right: 24px;
+}
+
+/* project */
+
+.box1 {

```

Comment:
<img width="346" alt="2" src="https://github.com/Nomarg/My-Portfolio/assets/77843050/38d42556-6129-4d8c-8461-a1177d812969">

- [ ] As said by the previous reviewer, you should wrap  all your project cards in a common container and use display:grip property to align its content(projects)

Hint 

```
 <section display:grip>
    `box1 display:flex`
    `box1`
<section/>

```